lib/src/utils/file_utils.dart

```diff
@@ -0,0 +1,27 @@
+import 'dart:io';
+
+import 'package:file/local.dart';
+import 'package:glob/glob.dart';
+import 'package:path/path.dart';
+
+import 'exclude_utils.dart';
+
+Set<String> extractFilesFromFolders(
```

Comment:
@incendial please rename to extractDartFilesFromFolders

Reply:
Done.